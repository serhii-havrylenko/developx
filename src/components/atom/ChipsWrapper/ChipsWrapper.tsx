import styled from 'styled-components';

const ChipsWrapper = styled.div`
  display: flex;
  justify-content: left;
  align-items: center;
  flex-wrap: wrap;

  & > *:not(:first-child) {
    margin: 10px;
  }
`;

export default ChipsWrapper;
