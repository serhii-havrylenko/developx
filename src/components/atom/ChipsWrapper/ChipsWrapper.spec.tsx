import { shallow } from 'enzyme';
import * as React from 'react';

import ChipsWrapper from './ChipsWrapper';

describe('<ChipsWrapper /> tests', () => {
  test('should match snapshot', () => {
    const tree = shallow(<ChipsWrapper />);
    expect(tree).toMatchSnapshot();
  });
});
