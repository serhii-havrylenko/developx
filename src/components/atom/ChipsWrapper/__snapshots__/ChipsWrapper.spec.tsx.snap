// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChipsWrapper /> tests should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: left;
  -webkit-justify-content: left;
  -ms-flex-pack: left;
  justify-content: left;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c0 > *:not(:first-child) {
  margin: 10px;
}

<div
  className="c0"
/>
`;
